src/constants.js

```diff
@@ -28,18 +28,20 @@ export function prefixForGoogleMaps (alwaysIncludeGoogle) {
     : 'https://maps.google.com/'
 }
 
-export const titles = {
-  'apple-maps': 'Apple Maps',
-  'google-maps': 'Google Maps',
-  citymapper: 'Citymapper',
-  uber: 'Uber',
-  lyft: 'Lyft',
-  transit: 'The Transit App',
-  waze: 'Waze',
-  yandex: 'Yandex.Navi',
-  moovit: 'Moovit',
-  'yandex-maps': 'Yandex Maps',
-  kakaomap: 'Kakao Maps'
+export function generateTitles (titles) {
+  return Object.assign({
```

Comment:
Should we maybe use ES6 syntax here? Something like:

```js
{
    'one': 'value',
    'other': 'value',
    ...(titles || {})
}
```

Reply:
OK, fixed. :)